cmd/eth/balance.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math/big"
+
+	"github.com/newalchemylimited/seth"
+	"github.com/newalchemylimited/seth/cc"
+)
+
+var cmdbal = cmd{
+	desc: "get the balance of an account",
+	do:   getbal,
+	fs:   flag.NewFlagSet("balance", flag.ExitOnError),
```

Comment:
Looks like a `flag.FlagSet` can be embedded and initialized with `flag.FlagSet.Init` in the same way that `flag.NewFlagSet` initializes it.

Reply:
done